DeeployTest/testUtils/typeMapping.py

```diff
@@ -33,6 +33,19 @@
 
 offsetType = namedtuple("offsetType", ("type", "offset"))
 
+_ALL_DTYPES = {t.typeName: t for t in (*IntegerDataTypes, *FloatDataTypes)}
+
+
+def parseDataType(name: str):
+    """
+    Parses a data type from its name.
+    :param name: The name of the data type.
+    :return: The corresponding data type class.
+    """
```

Comment:
Very minor, but we use the NumPy-style documentation.

```suggestion
    """Parses a data type from its name.

    Parameters
    ----------
    name : str
        The name of the data type.

    Returns
    -------
    class
        The corresponding data type class.

    Raises
    ------
    ValueError
        If the provided data type name is unknown.
    """
```

Reply:
I aligned it in [8128a14](https://github.com/pulp-platform/Deeploy/pull/68/commits/8128a142202551f77a10afccd90ea4b22a44d7be)